runtime/src/snapshot_config.rs

```diff
@@ -67,6 +70,7 @@ impl Default for SnapshotConfig {
                 snapshot_utils::DEFAULT_MAX_INCREMENTAL_SNAPSHOT_ARCHIVES_TO_RETAIN,
             accounts_hash_debug_verify: false,
             packager_thread_niceness_adj: 0,
+            zstd_compression_level: 0, // a level of 0 uses zstd's default
```

Comment:
It seems that this option is only applicable for archive_format = Zstd. Perhaps, we can roll this option into "ArchiveFormat" struct?